### PR TITLE
Demo

### PR DESCRIPTION
**Pull request by bengigo**
request to merge the first draft of mobile version portfolio

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
+# Micronaut Portfolio
 
-> Description the project.
+> This project is created as part of the Microverse curriculum practice. The purpuse of the project is to practice
+> *using HTML&CSS Linters*
+> *parsing a Figma design
+> *understanding Flexbox
+> *using images, media and backgrounds
+> *creating a personal portfolio
+> *while presenting a professional repository
```
Comment (gemmen29) [inline README.md:12]: Please add an image for your project in your README file, so the visitor of the repo can take a closer look at the website

File: index.html [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+<body>
+    
+    <!--header bar-->
+
+    <div class="container">
+        <header>
+            <div class="header">
+                <div class="header-div"><span id="logo">Bengi Genç</span></div>
+                <div class="header-div" id="burger"><img src="student-portfolio-img/burger-menu.png" alt="menu"></div>
+            </div>
```
Comment (gemmen29) [inline index.html:24]: Please wrap this part with a ` nav ` tag for semantic purposes

```
<header>
   <nav>
      ... content(Which in your case the div that take class header)
   </nav>
</header>
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+                        <div><input type="image" src="student-portfolio-img/I.png" alt="social"></div>
+                    </div>
+                    <div class="n"><input type="image" src="student-portfolio-img/n.png" alt="social"></div>
+                </div>
+
+                <div class="social-logo" id="social-3"><input type="image" src="student-portfolio-img/m.svg" alt="social"></div>
+
+                <div class="social-logo" id="social-4"><input type="image" src="student-portfolio-img/social-cat.svg" alt="social"></div>
+
+                <div class="social-logo" id="social-5"><input type="image" src="student-portfolio-img/peace-hand.svg" alt="social"></div>
```
Comment (gemmen29) [inline index.html:55]: Kindly wrap all social media icons to be on a list using ` ul ` containing multiple ` li ` for each social media. And each ` li `contains ` a ` tag that contains the social media icon.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @bengigo, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -16,9 +16,9 @@
 - HTML
 - CSS
 
-<!--## Live Demo (if available)-->
+## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://rawcdn.githack.com/bengigo/micronaut-portfolio/6969c1e16d92542ea8c033bc817871bf58013dd3/index.html)
```
Comment (Kingstalux) [inline README.md]: - [ ] (OPTIONAL) You can add a screenshot of your project directly on your README file and save the live demo for a link to your project which you do not currently have 👍

**Review by Kingstalux — APPROVED** [review]
Hi @bengigo , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

_check below for optional suggestions_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**